tools/executors/translate/index.ts

```diff
@@ -31,77 +23,49 @@ export default async function translateExecutor(
       context.workspace.projects[context.projectName].sourceRoot,
       'lib'
     );
-    options.path = options.path.map((p) => p + TYPE_SCRIPT_FILE_EXTENSION);
-    options.output = join(projectLibPath, options.output);
+    const jsonFilePaths = readdirSync(projectLibPath).filter((p) =>
```

Comment:
We use the glob package

Reply:
Is it really needed? We can use `readdirSync`, right? It's serving the purpose.

Reply:
We already using it, and it's cleaner